src/types/peer.rs

```diff
@@ -0,0 +1,66 @@
+use ethereum_types::U256;
+use serde_derive::{Deserialize, Serialize};
+
+use std::fmt;
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct PeerType {
+    pub active: usize,
+    pub connected: usize,
+    pub max: u32,
+    pub peers: Vec<PeerInfo>,
+}
+
+impl fmt::Debug for PeerType {
```

Comment:
Why not `#[derive(Debug)]`?

Reply:
older version of nightly gave error that Debug needed to be impl for PeerType. upgraded to latest nightly and error went away. switched to ``#[derive(Debug)]`` and removed debug impl.